src/NUnitFramework/tests/Constraints/UniqueItemsConstraintTests.cs

```diff
@@ -235,7 +235,7 @@ public void SlowPath_TakenWhenSpecialTypes(IEnumerable<object> testData)
             constraint.ApplyTo(items);
             stopwatch.Stop();
 
-            Assert.That(stopwatch.ElapsedMilliseconds, Is.GreaterThanOrEqualTo(30));
+            Assume.That(stopwatch.ElapsedMilliseconds, Is.GreaterThanOrEqualTo(30));
```

Comment:
why?

Reply:
I think it ran faster than this on a build.
A test that assumes it runs at least a certain time will fail when run on a faster machine.
We have a few tests assuming time. The ones that expect a maximum time seem to fail regularly on MacOs.

I agree however that this change has nothing to do with this PR and should not be part of it.